src/opencost_parquet_exporter.py

```diff
@@ -85,12 +98,20 @@ def get_config(
             datetime.now() - timedelta(1), '%Y-%m-%d')
         window_start = yesterday+'T00:00:00Z'
         window_end = yesterday+'T23:59:59Z'
+    if parquet_prefix is None:
+      window = datetime.strptime(window_start, "%Y-%m-%dT%H:%M:%SZ")
+      parquet_prefix = os.environ.get(
+        'OPENCOST_PARQUET_PREFIX',
+        f"{file_key_prefix}/year={window.year}/month={window.month}/day={window.day}"
```

Comment:
just remember to change the window here to the same variable you change above. 

This looks ok, it would respecgt the window_start for backfills.

Reply:
Not sure to understand why should I do here 🤷🏼 
I tried to keep backward compatibility 🤞🏼